src/heads-up/components/app-sidebar.vue

```diff
@@ -95,4 +105,30 @@ export default {
     height: 1em;
     fill: currentColor;
   }
+
+  .sidebar__socials {
+    margin-bottom: 20px;
+  }
+
+  .sidebar__footer {
+    margin-top: auto;
+    padding: 10px;
+    z-index: 10;
+
+    color: var(--label-color);
+    text-align: center;
+    background-color: var(--toolbar-bg-color);
+  }
+
+  .sidebar__footer-anchor {
```

Comment:
In the codebase the name `anchor` isn't used, but the name link is.
I think it's more consistent to use `.sidebar__footer-link`.